src/predictor/predict_fn.h

```diff
@@ -7,20 +7,26 @@
 #include "xgboost/tree_model.h"
 
 namespace xgboost::predictor {
+/** @brief Whether it should traverse to the left branch of a tree. */
+template <bool has_categorical>
+inline XGBOOST_DEVICE bool GetDecision(RegTree::Node const &node, bst_node_t nid, float fvalue,
```

Comment:
No need for `inline`. template function is inlined.

Reply:
Done.